### PR TITLE
Add Doxygen link to menu for SimpleITK 2.0

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Comment by blowekamp** [comment]
The 2.0 Doxygen will be added in a separate PR.

File: about.html [hunk 1]
```diff
@@ -114,6 +114,7 @@
           API<span data-feather="chevron-down" class="ml-1"></span>
         </a>
                     <div class="dropdown-menu px-3 py-4" aria-labelledby="navbarDropdown">
+                        <a class="dropdown-item pl-2 text-secondary" href="doxygen/v1_2/html/" target="_blank">Version 1.2</a>
                         <a class="dropdown-item pl-2 text-secondary" href="doxygen/v1_2/html/" target="_blank">Version 1.2</a>
```
Comment (zivy) [inline about.html:119]: Line is incorrect refers to 1.2, copy correct line from the updated index.html.

File: privacy_policy.html [hunk 2]
```diff
@@ -114,6 +114,7 @@
           API<span data-feather="chevron-down" class="ml-1"></span>
         </a>
                     <div class="dropdown-menu px-3 py-4" aria-labelledby="navbarDropdown">
+                        <a class="dropdown-item pl-2 text-secondary" href="doxygen/v1_2/html/" target="_blank">Version 1.2</a>
                         <a class="dropdown-item pl-2 text-secondary" href="doxygen/v1_2/html/" target="_blank">Version 1.2</a>
```
Comment (zivy) [inline privacy_policy.html:119]: Line is incorrect refers to 1.2, copy correct line from the updated index.html.

**Review by zivy — CHANGES_REQUESTED** [review]

**Comment by blowekamp** [comment]
Thanks for the catch.

**Review by zivy — APPROVED** [review]

**Pull request merged**